debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleConnection.java

```diff
@@ -101,6 +106,60 @@ public void resetSessionToCdb() {
         }
     }
 
+    public OracleDatabaseVersion getOracleVersion() {
```

Comment:
Is there a specific reason for doing this lazily? If not, let's do this in the constructor and make `databaseVersion` final.

Reply:
No specific reason, I'll move it.